packages/hint/src/lib/utils/resource-loader.ts

```diff
@@ -260,7 +260,7 @@ const getResource = (source: string, type: ResourceType, name: string) => {
  */
 const generateConfigPathsToResources = (configurations: string[], name: string, type: ResourceType) => {
     return configurations.reduce((total: string[], configuration: string) => {
-        const basePackagePaths = ['@hint/configuration-', 'hint-configuration-'];
+        const basePackagePaths = ['@hint/configuration-', 'webhint-configuration-', ''];
```

Comment:
you need to change how you call to `generateConfigPathsToResources` in order to make the empty string works, otherwise, it will not work as we expected.